### DISCUSSION TITLE
Iphone still dropping audio

### DISCUSSION THREAD

**Discussion by Charlesc22** [mozilla/hubs#2854] [General Questions & Discussion] [answered]
Hey @Liv, are you still having trouble with the audio on your Iphone? We still seem to have the same issue where the audio drops after about 2 minutes. I thought the Media Soup transition would have solved that issue. but it has not.

I understand that working with the Iphone is problematic, but could you guys do some sort of "Reconnect Audio" button for Iphone users that would reset the audio channel?
That could be a simple fix for now till a better option comes up?

Thanks

**Comment by emclaren** [comment, marked as answer]
Hi Charles, the best place to make note of this would be in the issue you opened #2445

We've also started tracking bugs with WebRTC with this form, so if you notice other issues with WebRTC please report them here so we can figure out whats causing it (might not be necessary for the iPhone issue, because this seems reproducible based on the information provided in the github issue)

https://docs.google.com/forms/d/e/1FAIpQLSd_dw8uQN2VjxLcxXmXFzzaJVU8IWXuUAnRJd1kjoB_TohqsA/viewform
